tests/data/modules_v2/complex_module_dependencies_mod1/setup.cfg

```diff
@@ -0,0 +1,14 @@
+[metadata]
+name = inmanta-module-complex-module-dependencies-mod1
+version = 1.1.1
+description = A module with an internal circular dependency between _init.cf and submod.cf and an external dependency on complex-module-dependencies-mod2.
+license = Apache 2.0
+author = Inmanta <code@inmanta.com>
+
+[options]
+zip_safe=False
+include_package_data=True
+packages=find_namespace:
```

Comment:
Requirements are  missing here. I see why you would leave them out, you probably want to test the recursive loading logic based on discovered imports instead of having all dependencies pulled in automatically when installing this module? In any case, could you add the reason as a comment so it's clear this module config is left incomplete intentionally?

Reply:
I added a comment in the `setup.cfg`